flask/app/families.py

```diff
@@ -114,13 +128,19 @@ def get_family(id: int):
         [
             {
                 **asdict(family),
+                "updated_by": family.updated_by.username,
+                "created_by": family.created_by.username,
                 "participants": [
                     {
                         **asdict(participants),
```

Comment:
Can you change this to be in the singular?

Reply:
Sorry Delvin, I didn't mean the name of the of the list, but the variable being used in the list `**asdict(participants)` !